_episodes/interviews/2022-07-15-tyler-mandry.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Asynchronous Rust with Tyler Mandry"
+date: 2022-07-16T13:50:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-07-15-tyler-mandry.mp3
+duration: "1:01:30"
+length: "59048856"
```

Comment:
```suggestion
duration: "0:56:49"
length: "54559137"
```
These are the values for the file I got from you @HangingClowns — did you use incorrect values accidentally, or do I have the wrong/an old file?